Source/Scene/FeatureTable.js

```diff
@@ -146,6 +146,83 @@ FeatureTable.prototype.hasProperty = function (index, propertyId) {
   return false;
 };
 
+/**
+ * Returns whether the feature has a property with the given semantic.
```

Comment:
I would  add a sentence in these semantic-related functions that this is used for `EXT_feature_metadata`, but not the older batch table.